src/utils.js

```diff
@@ -15,7 +15,7 @@ const LOGFILE_REGEX = /InVEST-natcap\.invest\.[a-zA-Z._]+-log-[0-9]{4}-[0-9]{2}-
  * @param  {string} jobDatabasePath - path to a json file with jobs metadata.
  * @returns {Promise} - Resolves sorted array of jobs with metadata.
  */
-export function loadRecentSessions(jobDatabasePath) {
+export function loadRecentJobs(jobDatabasePath) {
   return new Promise((resolve) => {
     const db = JSON.parse(fs.readFileSync(jobDatabasePath, 'utf8'));
     const sortedJobs = Object.entries(db).sort(
```

Comment:
I haven't given this much thought or attention yet, meaning we're probably relying on defaults in many places. 

I think the form elements including text inputs send strings with encoding inherited from the html document. HTML5 defaults to UTF-8. 

It sounds like node likes things to be explicit, but also ends up assuming UTF-8 if it's not. 
https://nodejs.org/api/stream.html#stream_readable_setencoding_encoding
>By default, no encoding is assigned and stream data will be returned as Buffer objects. Setting an encoding causes the stream data to be returned as strings of the specified encoding rather than as Buffer objects. 

https://nodejs.org/docs/latest-v12.x/api/buffer.html#buffer_buffers_and_character_encodings
> When converting between Buffers and strings, a character encoding may be specified. If no character encoding is specified, UTF-8 will be used as the default.